src/engine/enginemixer.cpp

```diff
@@ -285,11 +246,11 @@ void EngineMixer::processChannels(int iBufferSize) {
     EngineChannel* pLeaderChannel = m_pEngineSync->getLeaderChannel();
     // Reserve the first place for the main channel which
     // should be processed first
-    m_activeChannels.append(NULL);
+    m_activeChannels.append(nullptr);
     int activeChannelsStartIndex = 1; // Nothing at 0 yet
     for (int i = 0; i < m_channels.size(); ++i) {
-        ChannelInfo* pChannelInfo = m_channels[i];
-        EngineChannel* pChannel = pChannelInfo->m_pChannel;
+        ChannelInfo* pChannelInfo = m_channels[i].get();
```

Comment:
unfortunately not because its used below for indexing info `m_channelMainGainCache` on line 273 and 291. This could be solved by a C++23 `ranges::zip_view`...